Source/WebCore/css/MutableStyleProperties.cpp

```diff
@@ -1,5 +1,6 @@
 /**
  * Copyright (C) 2023 Apple Inc. All rights reserved.
+ * Copyright (C) 2014 Apple Inc. All rights reserved.
```

Comment:
It's not correct to add this. You can change the line above to 2014-2023 if it's wrong.

Reply:
It was meant to be Google. Fixed it in new push. :-)